src/static/riot/competitions/detail/submission_modal.tag

```diff
@@ -1,177 +1,125 @@
 <submission-modal>
-    <div class="ui {three: submission.admin}{two: !submission.admin} item small green secondary pointing menu">
-        <div class="active item" data-tab="downloads">Downloads</div>
-        <div class="item" data-tab="logs">Logs</div>
-        <div class="item" data-tab="admin" if="{submission.admin}">Admin</div>
+    <div class="ui large green pointing menu">
+        <div class="active item" data-tab="downloads">DOWNLOADS</div>
+        <div class="item" data-tab="logs">LOGS</div>
+        <div class="item" data-tab="admin" if="{submission.admin}">ADMIN</div>
     </div>
-    <div class="ui tab active" data-tab="downloads">
+    <div class="ui tab active modal-tab" data-tab="downloads">
         <div class="ui relaxed centered grid">
-            <div class="ui fourteen wide column">
-                <div class="ui list">
-                    <div class="item">
-                        <a href="{ submission.data_file }">My Submission</a>
-                    </div>
-                    <div class="item">
-                        <a href="{ submission.result }">Output from prediction step</a>
-                    </div>
-                    <div class="item">
-                        <a href="#">Output from scoring step</a>
-                    </div>
-                    <div class="item">
-                        <a href="#">Private output from scoring step</a>
-                    </div>
-                </div>
+            <div class="ui fifteen wide column">
+                <div class="ui horizontal divider"></div>
+                <table class="ui very basic table" id="downloads">
+                    <thead>
+                    <tr>
+                        <th><i class="download icon"></i> Files</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <tr>
+                        <td class="selectable file-download"><a href="{ submission.data_file }"><i class="file archive outline icon"></i> Submission File</a></td>
+                    </tr>
+                    <tr>
+                        <td class="selectable file-download"><a href="{ submission.result }"><i class="file outline icon"></i>Output from prediction step</a></td>
+                    </tr>
+                    <tr>
+                        <td class="selectable file-download"><a href="#"><i class="file outline icon"></i>Output from scoring step</a></td>
```

Comment:
do these need to be `a` tags?

Reply:
Yea, for styling, they need to be a tags